zoom doc update (#2262)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/webhook-sources/zoom.md

---
id: zoom
title: Zoom Source
description: Set up a Zoom HTTP Source as an endpoint for receiving Webhook events.
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/integrations/saas-cloud/zoom.png')} alt="Thumbnail icon" width="50"/><br/>

Zoom HTTP Source is an endpoint for receiving Webhook events that are grouped into the following core event types:
- Meeting Events
- Webinar Events
- Recording Events
- Zoom Room Events
- User Events
- Account Events

> For more information on Zoom Webhook events, see this [Zoom web page](https://marketplace.zoom.us/docs/api-reference/webhook-reference).

Reviewer: ```suggestion
For more information, see the [Zoom webhook events documentation](https://marketplace.zoom.us/docs/api-reference/webhook-reference).
```
